src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/role-detail.js

```diff
@@ -106,6 +107,7 @@ angular.module('platformWebApp').controller('platformWebApp.roleDetailController
         });
         blade.selectedAll = false;
     }
+    $scope.selectButtonClick = (row) => row.$selected = !row.$selected ;
```

Comment:
Could this be done in template, not in the controller?

Reply:
done